### DISCUSSION TITLE
Snowfake DB Connection – Search Path

### DISCUSSION THREAD

**Discussion by KevinWellsMLC** [dbeaver/dbeaver#12502] [Q&A] [answered]
System information:
Windows 2012 R2
DBeaver 7.2.4

Connection specification:
Database: Snowflake  5.3.0

I'm connecting to a Snowflake data base.

In the connection settings I have set the Schema:   MLC_EDW_CUST    ( my 1st schema of choice ).    However;  I also want to a include a 2nd level choice ( MLC_P0141 ) so I don't have to qualify my query's with <database_name>.<schema_name>.<object_name>

Is there a way to set the Snowflake DB “search path”  within the DBeavers connection settings rather than having to do an ‘Alter Session set search_path=’$current, …..‘ after I connect.

Thank you.

**Comment by serge-rider** [comment, marked as answer]
However you can configure it in the bootstrap queries dialog (on the connection edit dialog):

![image](https://user-images.githubusercontent.com/6398845/107514092-3b6cb080-6bba-11eb-9543-c1d7c35d2b29.png)
